src/jsutils/invariant.d.ts

```diff
@@ -0,0 +1 @@
+export function invariant(condition: unknown, message?: string): void;
```

Comment:
Can you please update the return type to `: asserts condition`?
Please also do this change in TS migration PR as it should remove a bunch of TS errors.